apps/web/src/styles/main.scss

```diff
@@ -24,6 +24,7 @@
 
 // 7 - COMPONENTS
 @use "7-components/InfoBox/InfoBox" as InfoBox;
+@use "7-components/BlockLink/BlockLink" as BlockLink;
```

Comment:
I don't think we need this asa we can do it with a btn group like this:

```njk
<div class="govuk-button-group">
	{{ govukButton({ text: text | default('Continue') }) }}
	<a class="govuk-link" href="appeal-review">Change outcome</a>
</div>
```

Reply:
Thanks, I tried to figure out how to get the correct styling but I didn't look at the button group

Reply:
I've removed this now